src/Select.tsx

```diff
@@ -452,8 +452,11 @@ const Select = React.forwardRef<BaseSelectRef, SelectProps<any, DefaultOptionTyp
     // =========================== Change ===========================
     const triggerChange = (values: DraftValueType) => {
       const labeledValues = convert2LabelValues(values);
-      setInternalValue(labeledValues);
 
+      if(!('value' in props)){
```

Comment:
`value={undefined}` is a valid uncontrolled mode. Should not deps on the prop `in` props.